pc_setup_user.sh

```
@@ -1,7 +1,7 @@
 cd
 mkdir git
 cd git
-git clone https://github.com/una-auxme/paf23.git
+git clone https://github.com/una-auxme/paf.git
 
-cd paf23
+cd paf
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)